docs/_data/prague-2021-config.yaml

```diff
@@ -0,0 +1,172 @@
+name: Prague
+
+shortcode: prague
+year: 2021
+city: Prague
+local_area: Czech Republic
+area: Europe
+area_adj: European
+tz: CEST
+email: prague@writethedocs.org
+color: gold
+rgb: 253,185,19
+hex: fdb913
+
+photos:
+  default: _static/conf/images/headers/prague-group.png
+
+buttons:
+  top:
+    - text: Welcome to our conference
+      link: /news/welcome
+#    - text: Date and ticket information
+#      link: /news/announcing-virtual-conf-dates-tickets
+    # - text: Read our conference guide
+    #   link: /welcome-wagon
+    # - text: See the Schedule!
+    #   link: /schedule
+#    - text: Buy a Ticket!
+#      link: /tickets
+    - text: Sponsor us
+      link: /sponsors/prospectus
+    #- text: Submit a Talk
+    #  link: /cfp
+    #- text: See the talks!
+    #   link: /speakers
+    # - text: See the Schedule!
+    #   link: /schedule
+    # - text: Watch the Live Stream
+    #   link: /livestream
+    # - text: Browse the photos
+    #   link_absolute: https://www.flickr.com/photos/writethedocs/albums/72157691507514803
+    # - text: Watch the videos!
+    #   link_absolute: https://www.youtube.com/playlist?list=PLZAeFn6dfHpkBJAPYFrob6gqdiBuePwGJ
+  bottom:
+     - text: Welcome to our conference
+       link: /news/welcome
+    # - text: See the Schedule!
+    #   link: /schedule
+#     - text: Date and ticket information
+#       link: /news/announcing-virtual-conf-dates-tickets
+     - text: Sponsor the conference
+       link: /sponsors/prospectus
+    # - text: See the talks!
+    #   link: /speakers
+#     - text: Buy a Ticket!
+#       link: /tickets
+    # - text: Prepare with the Welcome Wagon
+    #   link: /welcome-wagon
+    # - text: See the schedule!
+    #   link: /schedule
+    # - text: Watch the Live Stream
+    #   link: /livestream
+    # - text: Browse the photos
+    #   link_absolute: https://www.flickr.com/photos/writethedocs/albums/72157691507514803
+    # - text: Watch the videos!
+    #   link_absolute: https://www.youtube.com/playlist?list=PLZAeFn6dfHpkBJAPYFrob6gqdiBuePwGJ
+
+tickets:
```

Comment:
@ericholscher are we updating these prices?

Reply:
Set to 150/100/25 as discussed